src/main/java/graphql/schema/AsyncDataFetcher.java

```diff
@@ -64,7 +64,13 @@ public AsyncDataFetcher(DataFetcher<T> wrappedDataFetcher, Executor executor) {
 
     @Override
     public CompletableFuture<T> get(DataFetchingEnvironment environment) {
-        return CompletableFuture.supplyAsync(() -> wrappedDataFetcher.get(environment), executor);
+        return CompletableFuture.supplyAsync(() -> {
+            try {
+                return wrappedDataFetcher.get(environment);
+            } catch (Exception e) {
+                throw new RuntimeException(e);
```

Comment:
Should I maybe use a specific (internal) kind of exception here that can be unpacked by the error handler later?
Or perhaps sneaky-throw the original?